client/my-sites/earn/stats/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { Card } from '@automattic/components';
```

Comment:
For reviewers: None of the code in this file is new. It's all just moved over from memberships/index. As is, we're still showing the stats section on memberships/index as well, but I'd be planning to remove that in in a follow up PR.